cms/views.py

```diff
@@ -1,9 +1,31 @@
 from rest_framework import viewsets
 from rest_framework.decorators import action
 from rest_framework.response import Response
-from cms.models import Post
-from cms.serializers import PostSerializer
-from cms.permissions import UserPermission
+from cms.models import MenuItem, Post
+from cms.serializers import MenuItemShortSerializer, PostSerializer
+from cms.permissions import PostPermission
+
+
+class MenuItemViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    Obsluhuje API endpoint pre MenuItems
+    """
+    queryset = MenuItem.objects.order_by('-priority')
```

Comment:
nie je orderovanie tuto duplicitne? ked to uz je nastavene v Meta daneho modulu?

Reply:
Zistili sme že nie. Aspoň Tomáš vravel že kým to tam nebolo tak to neorderovalo